hudi-common/src/main/java/org/apache/hudi/common/fs/StorageSchemes.java

```diff
@@ -45,7 +45,9 @@
   // Aliyun OSS
   OSS("oss", false),
   // View FS for federated setups. If federating across cloud stores, then append support is false
-  VIEWFS("viewfs", true);
+  VIEWFS("viewfs", true),
+  //ALLUXIO
+  ALLUXIO("alluxio",true);
```

Comment:
Alluxio supports append? Thats a good surprise :)

Reply:
Alluxio does not support append :(